### PR TITLE
Update loadbalancer routes with tenant id

### PR DESCRIPTION
**Pull request by hegartyk**
Update loadbalancer routes with tenant id as a path parameter instead of a header.

### CODE REVIEW

File: pkg/api/v1/httptools.go [hunk 1]
```diff
@@ -25,6 +25,23 @@ func (r *Router) parseTenantID(c echo.Context) (string, error) {
...
+	}
+
+	if id != "" {
+		if _, err := uuid.Parse(id); err != nil {
+			return "", ErrInvalidUUID
+		}
+	}
+
+	return id, nil
+}
```
Comment (fishnix) [inline pkg/api/v1/httptools.go:43]: Did you give some thought to this function returning an error if the path param wasn't found?  You could `errors.Is()` in the calling function in that case.

**Review by fishnix — DISMISSED** [review]
I just had the one question, otherwise lgtm! 👍

File: pkg/api/v1/httptools.go (hunk 1 above)
Comment (hegartyk) [reply pkg/api/v1/httptools.go:44]: Yeah, that works. Updated 😄

**Review by andy-v-h — APPROVED** [review]
LGTM 💛

**Pull request merged**